include/circt/Conversion/ExportVerilog.h

```diff
@@ -13,12 +13,13 @@
 #ifndef CIRCT_TRANSLATION_EXPORTVERILOG_H
 #define CIRCT_TRANSLATION_EXPORTVERILOG_H
 
+#include "circt/Support/LoweringOptions.h"
```

Comment:
```suggestion
```

Reply:
Shoot, I had deleted that in the branch before the merge.